### ISSUE TITLE
[No prefix] when using --replay of json to generate sh script.

### ISSUE THREAD

**Issue by jtl999** [sahib/rmlint#305]
A sidenote: Sorry if this issue is user error, but I checked and no one else was in the #rmlint channel on Freenode so I thought I'd ask here.

rmlint --version
```
version 2.7.0 compiled: Sep  9 2018 at [14:00:20] "Toothless Taipan" (rev 017b007e)
compiled with: +mounts +nonstripped +fiemap +sha512 +bigfiles +intl +replay +xattr +btrfs-support
```

Run `rmlint -a sha256 --hidden -vvv -o json:rmlint.json` in a directory:

It works and `rmlint.json` contains valid information on duplicate files, etc.

Run `rmlint -a sha256 --hidden -vvv -o sh:rmlint.sh`:

`rmlint.sh` contains a valid script with much the same data.

Run `rmlint --replay rmlint.json -a sha256 --hidden -vvv -o sh:rmlint.sh`:

Many entries have `[nope: no prefix]` after them, and the `rmlint.sh` file doesn't list any files to remove.

Is this intended behavior?

**Comment by sahib** [comment]
Hello @jtl999,

you might have found a bug here.

First of all one thing you might not be aware about: ``--replay`` will only list the files under the paths specified in the ``--replay`` command. This enables you to have an incremental workflow, where you replay parts of the results by ``cd``'ing into subdirectories. 

In your case however, you did not specify any paths, which should cause ``rmlint`` to take the current working directory. Due to some internal logic fuckup, ``rmlint`` assumed ``rmlint.json`` to be the right (and only) path and did not add the current working directory as valid path.

I pushed a fix for this behaviour as of 0b903373. Please try it and see if it fixes your issue.

**Comment by jtl999** [comment]
Seems to work fine! Thanks.
